src/js/components/Save.tsx

```diff
@@ -4,12 +4,13 @@ import { useStore } from './StoreContext'
 import { observer } from 'mobx-react'
 
 export default observer(({ onSave }) => {
-  const { differentURL, tabMode } = useStore().AppStore
+  const { differentURL, tabMode, saved, draft } = useStore().AppStore
   return (
     <Button
       color='primary'
       disabled={differentURL && !tabMode}
       onClick={onSave}
+      style={(saved || draft) && { backgroundColor: '#f50057', color: 'white' }}
```

Comment:
Thanks, I'd prefer to use the Material UI native color props to control whether to highlight the Save button.

The code may look like this:
`color={(saved || draft) ? 'secondary' : 'default}`

https://material-ui.com/components/buttons/#text-buttons

Reply:
Thanks for the feedback, I'll update the PR with the suggested change.